src/components/commons/cards/CardCommonStyle.module.scss

```
@@ -0,0 +1,73 @@
+.card {
+  width: 100%;
+  background-color: $card-background;
+  border-radius: 0.8rem;
+
+  &-inner {
+    @include column-flexbox(start, start, $gap: 0.8rem);
```

```suggestion
    @include column-flexbox(start, start, 0.8rem);
```
$gap은 적지 않아도 되지 않을까요?